driver_test.go

```diff
@@ -1381,9 +1381,39 @@ func startDestroyInspectImage(t *testing.T, image string, taskName string) {
 	taskCfg := newTaskConfig(image, busyboxLongRunningCmd)
 	inspectData := startDestroyInspect(t, taskCfg, taskName)
 
-	parsedImage, err := parseImage(image)
+	d := podmanDriverHarness(t, nil)
+
+	imageID, err := getPodmanDriver(t, d).createImage(image)
 	require.NoError(t, err)
-	require.Equal(t, parsedImage, inspectData.Config.Image)
+	require.Equal(t, imageID, inspectData.Image)
+}
+
+func Test_createImage(t *testing.T) {
+	if !tu.IsCI() {
+		t.Parallel()
+	}
+
+	d := podmanDriverHarness(t, nil)
+	// Need to test: shortname, longname with/without transport
+	// TODO: Also test oci/docker archives
+	// TODO: Make sure that images do NOT exist before some tests
```

Comment:
This two TODO probably require adding  `api.ImageRemove` and `api.ImageSave` methods. Would it make sense to add them even if they are only used for the tests?